draft-smyslov-ipsecme-rfc8229bis.xml

```diff
@@ -710,21 +700,21 @@
           first over UDP before attempting over TCP.  If there is a response to the
           request sent over UDP, then the ESP packets should be sent directly over IP or over UDP port 4500 
           (depending on if a NAT was detected), regardless of if a connection on a previous
-          network was using TCP encapsulation.  If no response is received within some period of time after
-          several retransmissions, then the Initiator changes transport for this very exchange from
-          UDP to TCP and continues retransmitting. New INFORMATIONAL exchange MUST
+          network was using TCP encapsulation.  If no response is received within a certain period of time after
+          several retransmissions, the Initiator SHOULD change its transport for this exchange from
```

Comment:
I can make this a lower case "ought to"

Reply:
OK.

Reply:
I just noticed one more issue later in this sentence: "and continue retransmitting" is inconsistent with the restransmission behavior described above - in case of TCP no retransmission is needed. So it is better to rephrase "and resends a request message over new transport"? Meaning no continuous retransmissions, just a single resend over TCP. Or something like that, you probably can rephrase it better than I.